### PR TITLE
CIV-6214 dashboard status part2

### PR DESCRIPTION
**Pull request by TatsyMaslova**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-6214

### Change description ###
dashboard status part two.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by portizdev — DISMISSED** [review]

**Review by laarmada — DISMISSED** [review]

File: src/test/unit/common/models/dashboard/dashboardItem.test.ts [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import {DashboardClaimantItem, DashboardDefendantItem} from 'common/models/dashboard/dashboardItem';
+//import * as languageService from 'modules/i18n/languageService';
```
Comment (PaulH-HMCTS) [inline src/test/unit/common/models/dashboard/dashboardItem.test.ts:2]: Remove

Comment (TatsyMaslova) [reply src/test/unit/common/models/dashboard/dashboardItem.test.ts]: done 👍 thanks

**Review by PaulH-HMCTS — DISMISSED** [review]

File: src/main/views/features/breathingSpace/check-answers.njk [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+                                text: t('COMMON.BUTTONS.SUBMIT')
+                            }) }}
+                            {{ contactUsForHelp(t) }}
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </main>
+    </div>
 {% endblock %}
```
Comment (hmcts-version1-domini) [inline src/main/views/features/breathingSpace/check-answers.njk:36]: Please, leave a newline at the end.

**Review by hmcts-version1-domini — DISMISSED** [review]

File: src/main/views/features/breathingSpace/check-answers.njk (hunk 2 above)
Comment (TatsyMaslova) [reply src/main/views/features/breathingSpace/check-answers.njk]: not sure why this was formatted and changed by my pr... but i added an extra line as you said

**Review by PaulH-HMCTS — APPROVED** [review]

**Review by hmcts-version1-domini — APPROVED** [review]

**Pull request closed without merging**